### ISSUE TITLE
Feature request: Set Wallpaper

### ISSUE THREAD

**Issue by mikaellofgren** [swiftDialog/swiftDialog#453]
I want to be able to set the computer wallpaper when using Dialog for enrollment.
First use case is to be able to set a wallpaper to control the "blur" to an company branding color,
and when done set a company wallpaper.
And for easy implementation main screen should work just fine, just set and read current wallpaper.
I know about for example desktoppr but that's one extra app...
Thanks!

**Comment by Lotusshaney** [comment]
Maybe use this ? It works well

https://github.com/scriptingosx/desktoppr

**Comment by mikaellofgren** [comment]
Yes but as I wrote, one more extra app to deploy at enrollment...

**Comment by fraserhess** [comment]
I can't tell if the request is to set a wallpaper while Dialog is running (which seems in scope) or to set the wallpaper for a user's environment (which would seem to be out of scope).

If the latter, my approach is to lay the picture and configuration files down before the first user logs in
https://macadmin.fraserhess.com/2023/10/07/deploying-a-default-wallpaper-in-macos-sonoma/

**Comment by WardsParadox** [comment]
If you deliver assets via a package you can set the current users wallpaper using the JXA bridge before you run dialog.

**Comment by bartreardon** [comment]
using an image or colour (hue or solid) instead of the blur layer is a reasonable request and in scope for the kind of thing swiftDialog does. Setting wallpaper is not in scope though and as mentioned by others, there are better ways to do that.
I appreciate wanting to reduce the number of tools you need to deploy but this isn't the type of thing this particular tool is designed for.

**Comment by mikaellofgren** [comment]
And if I figure out a way to implement and make a pull request? Still out of scope?

**Comment by bartreardon** [comment]
still out of scope. if you want to set desktop wallpaper, that's thats desktoppr is for.

swiftDialog is for user notifications and interaction, not general purpose system configuration. many other tools use swiftdialog in their system configuration workflows but swiftialog's purpose is more general than that.

**Comment by mikaellofgren** [comment]
Alright, but I was almost done with some POC...
![wallpaper](https://github.com/user-attachments/assets/6bb176b3-e879-40f1-8880-8a1377ceb6b5)

**Comment by bartreardon** [comment]
just so I'm clear, I have no intention of adding "set wallpaper" functionality to swiftdialog. It's not that type of utility. 

Just because something is possible, doesn't mean it should be added in to an app. swiftDialog is _not_ an onboarding app. people use it for onboarding but that's not the only use case. You may as well ask to have wallpaper be added in to munki or jamf setup manager. 

tldr - setting wallpaper isn't happening.

**Comment by mikaellofgren** [comment]
I was just me trying out things, never mind, I somewhat also agree more or less.
